AnnoyancesFilter/sections/popups.txt

```diff
@@ -1524,8 +1513,6 @@ autoc-one.jp#$#body.modal-open { overflow: auto!important;padding-right: auto!im
 denverpost.com#$#body { overflow: auto !important; }
 denverpost.com#$#.modal-scrollable { display: none !important; }
 denverpost.com#$#.modal-scrollable ~ div.connext-modal-backdrop { display: none !important; }
-lostiempos.com,nazya.com,firstthings.com###colorbox
-lostiempos.com,nazya.com,firstthings.com###cboxOverlay
```

Comment:
I'mt not sure but think it's best to keep rules like these as are. These sites probably belong to the same group so it's convenient to place them side-by-side, in fact I kinda intentionally didn't merge some rules in JPF for similar reason.

Reply:
ok